src/data/clients.ts

```diff
@@ -930,5 +930,29 @@ export const Clients: Array<Client> = [
         url: 'https://tilo.dev/manet/'
       }
     ]
+  },
+  {
+    id: 'symfonium',
+    name: 'Symfonium',
+    description:
+      'An offline-first third-party music player that enhances your Jellyfin experience with streaming, syncing and full personalization',
+    clientType: ClientType.ThirdParty,
+    deviceTypes: [DeviceType.Mobile],
+    licenseType: LicenseType.Proprietary,
+    platforms: [Platform.Android],
+    primaryLinks: [
+      {
+        id: 'play-store',
+        name: 'Play Store',
+        url: 'https://play.google.com/store/apps/details?id=app.symfonik.music.player&referrer=utm_source%3Djellyfin%26utm_medium%3Ddownload'
```

Comment:
```suggestion
        url: 'https://play.google.com/store/apps/details?id=app.symfonik.music.player'
```

Reply:
This got discussed here: http://support.symfonium.app/t/tell-the-world-about-symfonium/5887/3?u=gamebeaker
![image](https://github.com/user-attachments/assets/03752b96-f6cb-450b-9258-4490d6834656)

Reply:
If you're convinced we need tracking links on our site I don't think this client is fitted to be listed.